packages/lit-html/README.md

```diff
@@ -1,14 +1,133 @@
-# lit-html
+# lit-html 2.0 Pre-release
 Efficient, Expressive, Extensible HTML templates in JavaScript
 
 [![Build Status](https://travis-ci.org/Polymer/lit-html.svg?branch=master)](https://travis-ci.org/Polymer/lit-html)
 [![Published on npm](https://img.shields.io/npm/v/lit-html.svg)](https://www.npmjs.com/package/lit-html)
 [![Join our Slack](https://img.shields.io/badge/slack-join%20chat-4a154b.svg)](https://www.polymer-project.org/slack-invite)
 [![Mentioned in Awesome lit-html](https://awesome.re/mentioned-badge.svg)](https://github.com/web-padawan/awesome-lit-html)
 
-## Experimental lit-html 2.0
+## 🚨 About this pre-release
+
+This is a major version pre-release of lit-html 2.0. See issue
+[#1182](https://github.com/Polymer/lit-html/issues/1182) for the full list of changes
+planned/considered for this release.
+
+This pre-release is not yet feature complete or API stable. Please note the
+breaking changes, known issues, and limitations below, and use at your risk
+until the stable release is available. Issues are welcome
+for unexpected changes not noted below or in the changelog.
+
+## 🚨 Breaking changes
+
+While `lit-html` 2.0 is intended to be a backward-compatible change for the
+majority of 1.x users, please be aware of the following notable breaking
+changes, and see the full [changelog](CHANGELOG.md#200---unreleased) for more
+details:
+  * New `directive` and `Part` API (see below for migration info)
+  * `render()` no longer clears its container on first render
+  * Custom `templateFactory`, `TemplateProcessor`, and custom tag functions are no
+    longer supported
+
+## 🚨 Known issues/limitations
+
+* **Browser support**: This pre-release should run on modern browsers, however a
+  change to factor legacy browser support (IE11, etc.) into an opt-in package is
+  ongoing. As such, this release will not run on some older browsers. This is a
+  temporary state.
+* **Limited directive implementation**: The following directives are not yet
+  implemented. This is a temporary state:
+  * `asyncAppend`
+  * `asyncReplace`
+  * `templateContent`
+  * `unsafeSvg`
+  * `until`
+
+## 🚨 Migrating directives
+
+While the API for _using_ directives should be 100% backward-compatible with
+1.x, there is a breaking change to how custom directives are _authored_. The API
+change improves ergonomics around making stateful directives while providing a
+clear pattern for SSR-compatible directives: only `render` will be called on the
+server, while `update` will not be.
+
```

Comment:
Add: `WARNING: The directive API is experimental and subject to change.`

Reply:
Done